hack/boilerplate.go.txt

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019. PingCAP, Inc.
+// Copyright YEAR PingCAP, Inc.
```

Comment:
- `.` is not needed, see https://github.com/pingcap/tidb/blob/master/cmd/importer/job.go#L1

Reply:
`YEAR` is a place holder, see example https://github.com/kubernetes/kubernetes/blob/master/hack/boilerplate/boilerplate.go.txt.